### ISSUE TITLE
Functions in Sphinx do not show "function"

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#17610]
See for instance https://docs.sympy.org/dev/modules/functions/special.html#sympy.functions.special.gamma_functions.digamma (c.f. https://github.com/sympy/sympy/issues/17596). 

The classes are listed like

```
class sympy.functions.special.gamma_functions.polygamma
```

but the functions are like

```
sympy.functions.special.gamma_functions.digamma(x)
```

The functions should also be 

```
function sympy.functions.special.gamma_functions.digamma(x)
```

I don't know how easy this is to fix. 

CC @sympy/gsod

**Comment by Lokesh2703** [comment]
Is there anything need to be done here. If yes, please reply and describe

**Comment by asmeurer** [comment]
`digamma` is a class now, but the problem is still there for functions. For instance, https://docs.sympy.org/dev/modules/simplify/simplify.html#sympy.simplify.simplify.simplify.

**Comment by Lokesh2703** [comment]
To mention the functions given above as classes, we need to convert them to classes. Do I need to convert them to classes? If yes, which functions are to be converted as classes?

**Comment by asmeurer** [comment]
No, they should not be converted to classes. This is about changing the formatting the Sphinx documentation (in the `doc/` directory).

**Comment by Lokesh2703** [comment]
Understood, can I work on this?

**Comment by Lokesh2703** [comment]
In sphinx, function is shown as `sympy.functions.special.gamma_functions.digamma(x)` by default, for it to be shown as `function sympy.functions.special.gamma_functions.digamma(x)` I could not understand what to change the formatting to, as the docstring is rendered by `:autofunction:` directive. If I change it to `:function:` it couldnot render the docstring.

**Comment by asmeurer** [comment]
Yeah this has to be changed in Sphinx itself somewhere. I don't know how to fix it. You'd have to figure that out (note that this issue is not tagged "easy to fix").
